### ISSUE TITLE
": No such file or directory php" after mounting of a local directus git repository

### ISSUE THREAD

**Issue by arboehme** [directus/directus-docker#35]
Hi,

if I try to mount my local directus git repository to the `/var/www/html` in the `docker-compose.yml`, the directus container goes in the loop with error `:No such file or directory php`  The line is logged out right after `Running directus!` 

If I comment out the volume mount, all works as expected. 

Here the corresponding line:

```
directus:
    volumes:
      - ${DIRECTUS_LOCAL}:/var/www/html
```

`DIRECTUS_LOCAL` is set to full-qualified path to the local directus git repository in the `.env` file in the same directory as the `docker-compose.yml` 

Do you have any idea, what I'm missing? 

Thank you and best regards,

**Comment by arboehme** [comment]
Hi,

after spending some hours digging in the build process of the directus docker container, I was able to resolve the issue. (Un-)Fortunately it was on my side.  :)

I found out, that the problem was the "hashbang"  `#!/usr/bin/env php` in the `bin/directus` script, which is called by `run-directus` command. 

The reason was the wrong line endings. Since I'am on Windows and my git client uses `core.autocrlf=true` by default, the checkout of directus repository automatically converted all LF in CRLF. 

After setting `core.eof=lf` and `core.autocrlf=inline` in my directus local repository and re-checkout all files, the docker-compose can now successfully start the directus container with mounted volume

Best regards
